pkg/services/ngalert/accesscontrol.go

```diff
@@ -0,0 +1,180 @@
+package ngalert
+
+import (
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/dashboards"
+	"github.com/grafana/grafana/pkg/services/datasources"
+)
+
+var (
+	rulesReaderRole = accesscontrol.RoleRegistration{
+		Role: accesscontrol.RoleDTO{
+			Name:        accesscontrol.FixedRolePrefix + "alerting.rules:reader",
```

Comment:
Does the syntax of the name need to follow the syntax of the actual actions? I was originally confused why we had two wordings `alerting.rules:reader`/`alert.rules:read`. Is this just a name, or will this be parsed by something?

Reply:
More specifically: should we consider aligning the name here with the spelling of the action?

Reply:
I follow the same rules that are used for other apis, where all roles are `*reader` or `*querier` etc.

Reply:
Names are actually keys. What user sees in UI is `DisplayName`